### PR TITLE
[OKTA-847065] Note to Postman Collections page

### PR DESCRIPTION
**Pull request by annejuan-okta**
## Description:
- **What's changed?**  Notes and updates link for the postman-collections file, remove from left-nav TOC
- **Is this PR related to a Monolith release?**  N/A

### Resolves:

* [OKTA-847065](https://oktainc.atlassian.net/browse/OKTA-847065)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/reference/postman-collections/index.md [hunk 1]
```diff
@@ -7,11 +7,14 @@ meta:
 
 # Import a Postman Collection
 
-Learn how to [use Postman with the Okta REST APIs](/docs/reference/rest/).
+**Note:** The following Postman collections are no longer kept up to date. For the most recent versions of the Postman collections, see [Okta Public API Collections](https://www.postman.com/okta-eng/okta-public-api-collections/overview).
```
Comment (barbaravo-okta) [inline packages/@okta/vuepress-site/docs/reference/postman-collections/index.md:10]:
```suggestion
**Note:** The following Postman collections are no longer maintained. For the most recent versions of the Postman collections, see [Okta Public API Collections](https://www.postman.com/okta-eng/okta-public-api-collections/overview).
```

File: packages/@okta/vuepress-site/docs/reference/rest/index.md [hunk 2]
```diff
@@ -62,7 +62,7 @@ You need a free Okta Developer Edition org to get started. Don't have one? [Crea
 
 Go to the Okta [Postman Collections](https://developer.okta.com/docs/reference/postman-collections/) reference to import the collection that you want to test.
 
-> **Note:** Alternatively, you can fork a collection from the [Okta Public API Collections](https://www.postman.com/okta-eng/workspace/okta-public-api-collections/overview).
+> **Note:** The Postman Collections reference is no longer kept up to date. For the latest version of the Postman collections, you can fork a collection from the [Okta Public API Collections](https://www.postman.com/okta-eng/workspace/okta-public-api-collections/overview).
```
Comment (barbaravo-okta) [inline packages/@okta/vuepress-site/docs/reference/rest/index.md:65]:
```suggestion
> **Note:** The Postman Collections reference is no longer maintained. For the latest version of the Postman collections, you can fork a collection from the [Okta Public API Collections](https://www.postman.com/okta-eng/workspace/okta-public-api-collections/overview).
```

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/postman-collections/index.md](https://github.com/okta/okta-developer-docs/blob/3dc887b415a0fc4668b49a0911d54865d20f21c0/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fpostman-collections%2Findex.md) | [80](https://okta.acrolinx.cloud/api/v1/checking/scorecards/9198c29d-2172-44f5-bc83-89d51d1836cc) | 60  | 8  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/rest/index.md](https://github.com/okta/okta-developer-docs/blob/3dc887b415a0fc4668b49a0911d54865d20f21c0/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Frest%2Findex.md) | [82](https://okta.acrolinx.cloud/api/v1/checking/scorecards/146b51d5-dae1-40b6-ad06-d6bb10ab986f) | 42  | 80  | 82  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.420cf2fe-744d-4a5c-8e6c-159a5c35fec5?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/postman-collections/index.md](https://github.com/okta/okta-developer-docs/blob/3ffc8b17db311699e18c87a62764fdca036311fb/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fpostman-collections%2Findex.md) | [80](https://okta.acrolinx.cloud/api/v1/checking/scorecards/8c6450fb-65ac-44e9-8e0a-fb416db19633) | 60  | 8  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/rest/index.md](https://github.com/okta/okta-developer-docs/blob/3ffc8b17db311699e18c87a62764fdca036311fb/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Frest%2Findex.md) | [82](https://okta.acrolinx.cloud/api/v1/checking/scorecards/9625a195-297d-4def-9890-9ba72a05fdb7) | 42  | 80  | 82  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.c89784bf-1961-42d1-a3b5-c3f5956a9cc2?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Review by barbaravo-okta — APPROVED** [review]

**Pull request merged**